pkg/virt-controller/watch/snapshot/restore.go

```diff
@@ -514,15 +540,29 @@ func (t *vmRestoreTarget) Reconcile() (bool, error) {
 	}
 	newVM.Annotations[lastRestoreAnnotation] = restoreID
 
-	_, err = t.controller.Client.VirtualMachine(newVM.Namespace).Update(newVM)
+	newVM, err = patchVM(newVM, t.vmRestore.Spec.Patches)
```

Comment:
do we need to do this patches also for an existing VM? if so why?

Reply:
Well, that's a good question. I've also asked myself that.

In practice I think the only "user" of the patching capability would be the cloning controller (which won't use patches for an existing VM). But, if someone does want to restore and make changes from the snapshot that was taken - why wouldn't we let them?

I think this capability can stay, but I won't insist on that.